build/tutorials/tutorials-contest/pangolin-token-subgraph/README.md

```diff
@@ -0,0 +1,480 @@
+
+Avalanche can best be described as a blockchain ecosystem and smart contracts platform that is built from the ground up to combat the notion that blockchains are inherently slow and not scalable. It does this through a novel consensus algorithm which is orders of magnitude more efficient and robust than Classical and Nakamoto consensus used in blockchains like Bitcoin and Ethereum. Avalanche boasts of higher throughput and lower latency when compared to other Layer 1 blockchains with about 4,500 transactions per second (tps) and an average time-to-finality of 2 seconds. It achieves these performance gains without sacrificing decentralization as its Proof-of-Stake network allows validators to stake 2,000 [AVAX](https://www.avax.network/individuals) (the native cryptocurrency of Avalanche) to secure the network using regular computer hardware as it is energy efficient and CPU optimized. Becoming a validator on Avalanche does not require specialized hardware and as a result, it has thousands of validators securing the network.
+
+The architecture of Avalanche is comprised of [3 main chains](https://docs.avax.network/learn/platform-overview) which derive their security guarantees from the [primary network](https://support.avax.network/en/articles/4135650-what-is-the-primary-network). The chains are the [Exchange Chain (X-Chain)](https://docs.avax.network/build/avalanchego-apis/exchange-chain-x-chain-api), the [Platform Chain (P-Chain)](https://docs.avax.network/build/avalanchego-apis/platform-chain-p-chain-api), and the [Contract Chain (C-Chain)](https://docs.avax.network/build/avalanchego-apis/contract-chain-c-chain-api). The Exchange Chain is used for creating and trading digital assets and relies on Avalanche Consensus Protocol, the Platform Chain is used for creating new [subnets](https://docs.avax.network/build/tutorials/platform/create-a-subnet) and custom blockchains. Subnets are a set of validators that can validate a set of blockchains because in Avalanche, a blockchain is validated by exactly one subnet but a subnet can validate several blockchains. The Contract Chain is an instance of the [Ethereum Virtual Machine (EVM)](https://ethereum.org/en/developers/docs/evm/) running on Avalanche. Both the Platform Chain and the Contract Chain use the [Snowman Consensus Protocol](https://docs.avax.network/#snowman-consensus-protocol). Avalanche, therefore supports Ethereum tooling and [Solidity](https://docs.soliditylang.org/en/v0.8.7/) as smart contracts built for Ethereum can be deployed on the Contract Chain with the added benefit of increased throughput. Decentralized applications can also be built natively for the Contract Chain using tools like [MetaMask](https://metamask.io/), [Truffle](https://www.trufflesuite.com/), [Waffle](https://getwaffle.io/), [Remix](https://remix.ethereum.org/). etc.
+
+Below is a diagram that further illustrates the points discussed.
+
+
+
+![alt_text](images/image2.png "Avalanche Architecture")
```

Comment:
Rename images and paths so that it's clear what tutorial each image is associated with. For example rename this image and path from `images/image2.png` to `images/pangolin-token-subgraph-image2.png`

Reply:
Images names have been changed to the naming convention.